code/obj/item/deployable_turret.dm

```diff
@@ -76,7 +76,6 @@ ABSTRACT_TYPE(/obj/item/turret_deployer)
 	desc = "A Nanotrasen Automatic Riot Control System Deployer. Use it in your hand to deploy."
 	icon_state = "st_deployer"
 	w_class = W_CLASS_BULKY
-	health = 125
```

Comment:
this changes the health of NARCS made from a turret deployer, since "health" is being used to store the health of the turret. Should just move to a different var tbh